docs/source/explanation/Main_algorithms/anomaly_classification.rst

```diff
@@ -0,0 +1,14 @@
+Anomaly classificaiton
+=========
+
+*********
+Dataset format
+*********
+
+*********
+Backbones
```

Comment:
Models? It sounds less domain-specific

Reply:
I think it is quite common name for models which performes feature extraction. For consistency I named this section as for other tasks.

Reply:
On the classification page in that section model templates are actually mentioned, which looks quite confusing:
```
*********
Backbones
*********

We support the following ready-to-use templates:
```
For classification the templates are synomical to backbones somehow, but for object detection each template represent a different algorithm.

Reply:
Okay, I've changed naming to "Models"